### ISSUE TITLE
Issue with e-mail invitations

### ISSUE THREAD

**Issue by mateoayat** [d4software/QueryTree#99]
While inviting a new user with the current develop branch, the following exception is thrown.

> An unhandled exception occurred while processing the request.
InvalidOperationException: The LINQ expression 'DbSet<ApplicationUser>
.Where(a => string.Compare(
strA: a.Email,
strB: __viewModel_Email_0,
ignoreCase: True) == 0)' could not be translated. Either rewrite the query in a form that can be translated, or switch to client evaluation explicitly by inserting a call to either AsEnumerable(), AsAsyncEnumerable(), ToList(), or ToListAsync(). See https://go.microsoft.com/fwlink/?linkid=2101038 for more information.

Is this caused by the update to .NET 3.1? Has anyone else had this issue so far?

**Comment by mateoayat** [comment]
I've confirmed that the issue arises from the newer .NET framework. Newer versions do not support the email evaluation as is currently written. I've fixed it by changing line 202 of TeamController.cs 
`var targettedUser = db.ApplicationUsers.FirstOrDefault(u => string.Compare(u.Email, viewModel.Email, true) == 0);`
to
`var targettedUser = db.ApplicationUsers.AsEnumerable().FirstOrDefault(u => string.Compare(u.Email, viewModel.Email, true) == 0);`
I've made a pull request in the develop branch for this quick patch, but I hope the solution helps anyone with the same issue.
